static/app/components/smartSearchBar/index.tsx

```diff
@@ -751,7 +755,13 @@ class SmartSearchBar extends React.Component<Props, State> {
     }
 
     return [
-      tagKeys.map(value => ({value, desc: value})),
+      tagKeys
+        .map(value => ({
+          value,
+          desc: value,
+          documentation: getFieldDoc ? getFieldDoc(value.slice(0, -1)) : '',
+        }))
+        .sort((a, b) => a.value > b.value),
```

Comment:
Look like we use `localeCompare` for these kinds of sorts, does this work?
```suggestion
        .sort((a, b) => a.value.localeCompare(b.value),
```